python/pynessie/auth/aws.py

```diff
@@ -17,7 +17,7 @@
 """Use AWS4Auth and botocore to fetch credentials and sign requests."""
 from botocore.credentials import get_credentials
 from botocore.session import Session
-from requests_aws4auth import AWS4Auth
+from requests_aws4auth import AWS4Auth  # type: ignore
```

Comment:
How about ignoring `requests_aws4auth` and `marshmallow_oneofschema` in the config instead?

Reply:
as i mentioned before i based it on the amount of usage spots... since both are only imported in a single spot i ignored it locally.
if you insist we do all ignores via config file i can do that?

Reply:
IMHO I would prefer via the config file to keep things in a central place to manage. WDYT?

Reply:
ok done in latest commit